### PR TITLE
Fancy garbage flash with block preview

### PR DESCRIPTION
**Pull request by frostburn**

### CODE REVIEW

File: ui/fancy/src/css/style.less [hunk 1]
```diff
@@ -16,6 +16,21 @@ body {
...
+  to {
+    background-color: lighten(@color-garbage, 30%);
+  }
+}
+
+@keyframes garbage-preview {
+  from {
+    opacity: 0.5;
+  }
+  to {
```
Comment (RauliL) [inline ui/fancy/src/css/style.less]: Separate declarations from each other with blank line.

File: ui/fancy/src/js/garbage.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    const draw = SVG(slab.uuid);
+
+    for (let i = 0; i < slab.width * slab.height * 4; ++i) {
+      const color = new SVG.Color({
+        r: 60 + Math.floor(20 * rand()),
+        g: 50 + Math.floor(15 * rand()),
+        b: 40 + Math.floor(10 * rand()),
+      });
+      const circle = draw.circle(`${rand()}em`);
```
Comment (RauliL) [inline ui/fancy/src/js/garbage.js]: Missing blank line between local variable declarations and statements.

**Comment by RauliL** [comment]
Otherwise very good.

**Comment by frostburn** [comment]
Issues fixed. Merging.

**Pull request merged**